external/io_int/makefile

```diff
@@ -30,7 +30,7 @@ io_int.f: io_int.F90 module_internal_header_util.o
 	fi
 
 io_int.o: io_int.f ../../inc/intio_tags.h
-	$(FC) $(FCFLAGS) -I../../inc -I../ioapi_share -o $@ -c $*.f
+	$(FC) $(FCFLAGS) -I../../inc -I../../frame -I../ioapi_share -o $@ -c $*.f
```

Comment:
Add an explanation in the commit message as to why `-I../../frame` is required for the include files for CPP.

Reply:
First, this isn't a c preprocessor line, it's a compilation step. :)

2nd, I added this in a few places to resolve compiler errors similar to:

    PGF90-F-0004-Unable to open MODULE file module_internal_header_util.mod (diffwrf.F90: 51)

I also had to make this change to get the gnu compiler toolchain to work. If there's only one instance left, I suspect some of them were eliminated when I rebased.  I can go back and check to make sure this one is needed. This was a showstopper for me until I fixed it.

Reply:
Looks like this one wasn't necessary or isn't necessary anymore. Removing.

Reply:
The other two are not necessary either. It looks like the module files are now copied to the directory containing the code where they're used.